doc/pages/upgrade/030-upgrade_10.md

```diff
@@ -39,3 +39,23 @@ kubectl edit astarte -n astarte
 Find the `version` field in the Astarte Spec section and change it according to your needs. Once the
 yaml file will be saved, the Operator will take over ensuring the reconciliation of your Astarte
 instance to the requested version.
+
+### Caveats for Astarte Flow
+
+Currently, although Astarte Flow is a component of Astarte, it doesn't follow Astarte's release
```

Comment:
Please, add a link to Flow doc: `[Astarte Flow](https://docs.astarte-platform.org/flow/)`.